is_real_robot: true
device:
  name: "hopper_teststand"
  sensors:
    # joints data
    joint_positions:
      size: 2
    joint_velocities:
      size: 2
    joint_torques:
      size: 2
    joint_target_torques:
      size: 2

    # additional data
    contact_sensors:
      size: 1
    slider_positions:
      size: 1
    height_sensors:
      size: 1
    ati_force:
      size: 3
    ati_torque:
      size: 3
  controls:
    ctrl_joint_torques:
      size: 2
hardware_communication:
  max_missed_control: 10
  control_period: 1000000 # in nano second this means 1ms <=> 1kz

motor_KT: 0.025    # The Kt constant of the motor [Nm/A]: tau = I * Kt
